views/index.hbs

```diff
@@ -4,6 +4,11 @@
       <input id="select2" type="hidden"/>
       <div class="submit-section">
         <div class="btn btn-primary quiz-btn">Take Quiz</div>
+        <div>
+          <br />
+          Or challenge yourself or a friend to: <a href="#">Top PPG this season (NBA)</a>, <a href="#">Most TDs by tight ends (NFL)</a>, <a href="#">Most shut-outs by goalkeeper (EPL)</a>
```

Comment:
This is just placeholder right? We are gonna make a backend call to fetch relevant stuff to display here.

Also standard rosterblitz font is HelveticaNeue-Thin. I will make it a global. Should be used everywhere